driver-core/src/main/com/mongodb/client/model/TimeSeriesOptions.java

```diff
@@ -32,6 +32,7 @@
  * @mongodb.driver.manual core/timeseries-collections/ Time-series collections
  */
 public final class TimeSeriesOptions {
+    private static final String PARAMETER_TIME_UNIT = "timeUnit";
```

Comment:
The names of the `bucketMaxSpan`, `bucketRounding` parameters are duplicated in string literals in argument checks, but the name of the `timeUnit` parameter is static field to avoid duplication. This is not consistent.

P.S. I think, duplicating a parameter name in error messages across methods is always fine.